microsite/blog/2021-01-12-new-backstage-feature-kubernetes-for-service-owners.md

```diff
@@ -0,0 +1,63 @@
+
+## Automatic error reporting
+
+Instead of trying different kubectl commands to figure out where an error occurred, Backstage will automatically find and highlight errors in Kubernetes resources that are affecting your service. So you can spend time fixing errors, not hunting for them.
+
+## Autoscaling limits at a glance
+
+Backstage also shows you how close your service is to its autoscaling limit. Coming up to a period of high load? Now you will be able to see how your horizontal autoscaling is dealing with it across multiple clusters.
+
+## Pick a cloud, any Cloud
+
+Since Backstage communicates directly with the Kubernetes API, it’s cloud agnostic — it doesn’t matter how or where you’re running Kubernetes. You’ll always get the same familiar view of your deployments, whether you’re:
+
+- Deploying to clusters on AWS, Azure, GCP, or another cloud provider
+- Using an unmanaged or managed Kubernetes service (like OpenShift, etc.)
+- Migrating from one cloud provider or service to another
+- Testing on a single local machine or deploying to a dozen clusters in production
+
+In short: local or global, single or multi-cloud, managed or unmanaged — Backstage always provides a seamless Kubernetes experience for your service owners’ day-to-day development needs.
+
+## Rethinking the developer experience
+
+The philosophy behind Backstage is simple: improve developer experience by reducing infrastructure complexity. As popular and widespread as Kubernetes has become, all of the tools to date have been geared toward the needs of cluster admins. These tools add unnecessary complexity to the workflows of the typical developer building, testing, and deploying services.
+
+We believe Backstage Kubernetes gives developers back control of their services by providing a more focused and consistent experience. Backstage provides a single standard for developers to monitor their Kubernetes deployments, regardless of the underlying cloud infrastructure.
+
+## Future iterations
+
+The current focus of Kubernetes in Backstage is Deployments/ReplicaSets/Pods — but we know that not everyone utilizes these.
+
+As we continue to grow and develop Kubernetes in Backstage with the community, we hope to offer support for Kubernetes resources beyond Deployments and Custom Resource Definitions. You can browse or add open issues for the plugin [here].
+
+## Getting started
+
+We made the Kubernetes plugin a core feature of Backstage. Like Software Templates (scaffolder) and TechDocs, the k8s-plugin is installed with the core app. When you update the app to the latest version and go to the Kubernetes tab of any service, you will be asked to provide your cloud provider credentials. Learn more in the plugin README files for [kubernetes] and [kubernetes-backend].
```

Comment:
What's interesting, is that the doco at those plugin repos _duplicates_ that in https://backstage.io/docs/features/software-catalog/kubernetes-in-backstage. (understandly, I'd guess an older lift + shift from plugin to docs?) I just filed a PR coincidentally to improve that info in the docs in #4020 . Wonder if they should be aligned in some way, either removed from the docs and pointed to the plugins, or elevated FROM the plugins with a pointer back to the docs - esp if their usage is more positioned as a core feature rather than just a plugin.

Alternatively, if we remove from docs I could change my PR content to go back into the plugin README, but I think the doco is more friendly to an end-user and thus pointing plugin to the nice doco from the README might be preferred.

Reply:
Yes, we only merged that doc yesterday so I think this line is out of date

Reply:
We'll update this line to point to docs. Thank you @adamdmharvey!